src/pykeen/training/training_loop.py

```diff
@@ -341,84 +438,118 @@ def _train(  # noqa: C901
             num_workers=num_workers,
         )
 
+        # Save the time to track when the saved point was available
+        last_checkpoint = time.time()
+
         # Training Loop
         for epoch in epochs:
-            # Enforce training mode
```

Comment:
@lvermue thanks for taking care of merging the updates on master back into this, I would have done it for you but I didn't want to mess any of the new code up